spec/index.bs

```diff
@@ -1426,6 +1426,16 @@ applications should generally request the "worst" limits that work for their con
         {{GPURenderPassDescriptor}}.{{GPURenderPassDescriptor/colorAttachments}},
         and {{GPURenderPassLayout}}.{{GPURenderPassLayout/colorFormats}}.
 
+    <tr><td><dfn>maxColorAttachmentBytesPerPixel</dfn>
```

Comment:
nit: probably needs to take into account depth-stencil as well, and is per fragment, so `maxAttachmentBytesPerFragment`?

Reply:
Good point, the Metal wording "Maximum total render target size, per pixel, **when using multiple color render targets**" threw me off but it seems it should include depth-stencil. Fixed